features/area_page.feature

```diff
@@ -2,7 +2,7 @@
 # area type TOML table names in features/area_profiles_component.go areas.AreaDetails struct.
 
 Feature: Groups
-
+    @avoid
```

Comment:
Why are you disabling the tests? checked out the branch locally and the tests pass anyway.

Reply:
I was fixing the audit test fail, that occurred because of some vulnerability package install. Since, multiple test were failing that time, so I was trying to figure out individual audit test by disabling component test.